src/screens/Services/Services.js

```diff
@@ -164,6 +169,14 @@ class ServicesScreen extends React.Component<Props, State> {
         action: () => navigation.navigate(POOLTOGETHER_DASHBOARD),
       });
     }
+    services.push({
+      key: 'sablier',
+      title: 'Sablier money streaming',
+      body: 'Stream money to people and organizations in real-time with just one deposit',
+      disabled: SWServiceDisabled,
+      label: SWServiceLabel,
+      action: () => navigation.navigate(SABLIER_STREAMS),
+    });
```

Comment:
Could you please wrap this in `firebaseConfig.getBoolean('feature_services_sablier')` please? This will allow us to toggle this feature remotely.